### ISSUE TITLE
How do I generate a dataset for real 360?

### ISSUE THREAD

**Issue by guswhd** [google-research/jax3d#190]
First, I wanted to confirm that it worked well on the nerf synthetic dataset and then create a real 360 dataset and train it.
However, there is not much information, so I query https://github.com/google-research/jax3d/issues/175 After seeing this problem, I thought about dragging mip nerf's dataset to real 360 and downloading the dataset to learn. Remember, I tried to watch it and it didn't work out well.
To sum up, I want to train mobilenerf with videos I shot, but I'm not sure how to create a dataset. help!!

**Comment by kim-jinuk** [comment]
If you only have RGB images, using COLMAP and the link below will help you create a "poses_bounds.npy" file.

[https://github.com/Fyusion/LLFF/tree/master](url)

**Comment by guswhd** [comment]
> If you only have RGB images, using COLMAP and the link below will help you create a "poses_bounds.npy" file.
> 
> [https://github.com/Fyusion/LLFF/tree/master](url)

thank you !! 
Are there any data generation tips that might be helpful in learning nerf?

**Comment by DWhettam** [comment]
I'd recommend looking at thelocal_colmap_and_resize.sh script from here also: https://github.com/google-research/multinerf

I believe a combination of that script and the LLFF imgs2poses.py script is what is used to create your own dataset.
